src/states_screens/help_screen_5.cpp

```diff
@@ -0,0 +1,84 @@
+//  SuperTuxKart - a fun racing game with go-kart
+//  Copyright (C) 2009-2015 Marianne Gagnon
```

Comment:
These should be "Copyright (C) 2000-2016 The SuperTuxKart team and contributers".